person.rb

```diff
@@ -1,6 +1,7 @@
 require './nameable'
+require './rental'
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] According to the relation: `A person has many rentals`, Setters for `@rentals` should be present.